torch/_torch_docs.py

```diff
@@ -8930,6 +8930,54 @@ def merge_dicts(*dicts):
 """.format(**factory_common_args))
 
 
+add_docstr(torch.kaiser_window, """
+kaiser_window(window_length, beta, periodic=True, *, dtype=None, \
+layout=torch.strided, device=None, requires_grad=False) -> Tensor
+""" + r"""
+Kaiser window function.
+
+.. math::
+    w[n] = I_0 \left( \beta \sqrt{1 - \frac{4n^2}{(M-1)^2}} \right) / I_0( \beta )
+
+with
+
+.. math::
+    -\frac{M-1}{2} \leq n \leq \frac{M-1}{2}
+
+where :math:`M` is the full window size.
+
+If :attr:`periodic` is true, the :math:`M` in
+above formula is in fact :math:`\text{window\_length} + 1`. 
+This flag determines whether the returned window trims off 
+the last duplicate value from the symmetric window and is
+ready to be used as a periodic window with functions like
+:meth:`torch.stft`. Also, we always have
+``torch.kaiser_window(L, B, periodic=True)`` equal to
+``torch.kaiser_window(L + 1, B, periodic=False)[:-1])``.
+
+.. note::
+    If :attr:`window_length` :math:`=1`, the returned window contains a single value 1.
+
+""" + r"""
+Args:
+    window_length (int):  positive integer controlling the returned window size.
+    beta: shape parameter for the window
+    periodic (bool, optional): If True, returns a periodic window, for use in spectral analysis.
+        If False, returns a symmetric window, for use in filter design.
+
+Keyword args:
+    {dtype}
+    layout (:class:`torch.layout`, optional): the desired layout of returned window tensor. Only
+          ``torch.strided`` (dense layout) is supported.
+    {device}
+    {requires_grad}
+
+Returns:
+    Tensor: A 1-D tensor of size :math:`(\text{{window\_length}},)` containing samples from the window
```

Comment:
I don't think you need a "Returns:" here. The return type is clear. Typically we only include this if the return type is complicated or unexpected. For example, if the returned value is a tuple of two tensors.